### PR TITLE
Rearrange to prepare for more components

### PR DESCRIPTION
**Pull request by svanherk**
Figure it'd be easier to review if I do rearranging now, before adding any new logic.  Setting things up so additional components are grouped with the corresponding page.  I also planned to do the copy page and create page in the same component because they're the same route (`/create`).  But the data flow is a lot cleaner if they're broken into separate components, which I've done below.

### CODE REVIEW

File: src/create/copy-role-page-create.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import './copy-role-copy.js';
```
Comment (svanherk) [inline src/create/copy-role-page-create.js:1]: I also played around with adding this to the loader itself, which does work:
```
	{
		pattern: ['/', '/create'],
		view: (context) => {
			if (context.search.roleId) {
				return html`
					<d2l-copy-role-copy
						org-unit=${ifDefined(context.search.ou)}
						role-id=${context.search.roleId}>
					</d2l-copy-role-copy>`;
			} else {
				return html`
					<d2l-copy-role-create
						org-unit=${ifDefined(context.search.ou)}>
					</d2l-copy-role-create>`;
			}
		},
		loader: () => Promise.all([import('./src/create/copy-role-copy.js'), import('./src/create/copy-role-create.js')])
	}
```
If we did ever need to expand functionality though, might get too messy in the router.  It'd be pretty easy to switch back to a component if we did end up needing that though.  Thoughts?

Comment (dlockhart) [reply src/create/copy-role-page-create.js:1]: Could we split it out into 2 separate routes?

Comment (svanherk) [reply src/create/copy-role-page-create.js:1]: Like change the copy route from `/create?roleId=<whatever>` to something like `/copy?roleId=<whatever>`?  Yeah we should be able to do that, I _think_ this is the only place that calls it (although it's tricky to search for): https://search.d2l.dev/xref/lms/platformTools/security/static/edit_common.js?r=022ccd55#31.  We'd break bookmarks, but that's a weird bookmark to have

Comment (svanherk) [reply src/create/copy-role-page-create.js:1]: I could also add a simple redirect in the create page, saying "was I passed a roleId?  Send this over to `/copy`"

Comment (dlockhart) [reply src/create/copy-role-page-create.js:1]: Yeah something like that. Seems weird for the same page to be doing two things depending on a query-string param.

Comment (svanherk) [reply src/create/copy-role-page-create.js]: Yeah I like that, updated!

**Review by dlockhart — APPROVED** [review]

**Pull request merged**